### ISSUE TITLE
Errors Handling

### ISSUE THREAD

**Issue by Dinuz** [chnejohnson/vue-dapp#100]
I am able to handle errors (connections, chain change etc) but I am not able to prevent the error to popup in my console. So I always get a red error in console and browser also if the error is handled properly in my code.
Is this behavior by design?

For example how I can prevent errors like this:
index.esm.js:1547 Uncaught (in promise) Error: Error: unknown account #0 (operation="getAddress", code=UNSUPPORTED_OPERATION, version=providers/5.7.2)

to get written in console?

**Comment by chnejohnson** [comment]
@Dinuz Got it, there are some `console.error` in the package. Maybe I should use `throw new Error` to handle all these situations.

https://github.com/chnejohnson/vue-dapp/blob/7cf7bdda3b7ac1518c5a8fcbbc8c41d86de32225/src/composables/useEthers.ts#L104

But in the component like Board.vue, I think there is no way to catch the error even though I write `throw new Error`

https://github.com/chnejohnson/vue-dapp/blob/7cf7bdda3b7ac1518c5a8fcbbc8c41d86de32225/src/components/Board.vue#L47-L59

**Comment by chnejohnson** [comment]
I published a beta version `vue-dapp@0.6.3-beta.1` the codebase is on branch `error-handling` It removed all the `console.error` and you can handle connect error like the following code
```vue
 <vd-board
    v-if="connectorsCreated"
    :connectors="connectors"
    :connectErrorHandler="connectErrorHandler"
    dark
  >

<script>
const connectErrorHandler = (error: any) => {
  console.error('Connect error: ', error)
}
</script>
```

**Comment by Dinuz** [comment]
Gonna try it @chnejohnson thank you

**Comment by chnejohnson** [comment]
Fixed in v0.6.3
